lib/bolt/bolt_option_parser.rb

```diff
@@ -702,15 +714,8 @@ def initialize(options)
 
       separator "\nINVENTORY OPTIONS"
       define('-t', '--targets TARGETS',
-             'Identifies the targets of command.',
-             'Enter a comma-separated list of target URIs or group names.',
-             "Or read a target list from an input file '@<file>' or stdin '-'.",
-             'Example: --targets localhost,target_group,ssh://nix.com:23,winrm://windows.puppet.com',
-             'URI format is [protocol://]host[:port]',
-             "SSH is the default protocol; can be #{TRANSPORTS.keys.join(', ')}",
-             'For Windows targets, specify the winrm:// protocol if it has not be configured',
-             'For SSH, port defaults to `22`',
-             'For WinRM, port defaults to `5985` or `5986` based on the --[no-]ssl setting') do |targets|
+             'Specify the targets or group of targets to run on.',
+             "For more information, see 'bolt guide targets'.") do |targets|
```

Comment:
Aha, ok, I see now where the guide came from. It seems like all this text belongs in a docs section somewhere that we can link to from the guide.

Reply:
Most of this info is present on the 'Run Bolt' page: https://puppet.com/docs/bolt/latest/running_bolt_commands.html#specify-a-transport
I'm going to leave this in for now, but I'd like to revisit it later since this text is from the very early days of Bolt and there might be better ways to surface this info to users rather than put it in the description for the option.